chapters/10-community-involvement.Rmd

```diff
@@ -15,4 +15,37 @@
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+Complete these tasks for your package for the first project assignment. For more information and tips at any step, refer to the [Mozilla Open Leadership Training Series resources](https://mozilla.github.io/open-leadership-training-series/articles/readme/)
+
+1. Review your README
+ - Write a one-paragraph description of your package using the [Up-Goer Five Text Editor](https://splasho.com/upgoer5/). (You don't need to use this in your actual README, but it is a fun exercise to see what words you think are common might not be!)
+ - Review your README from the viewpoint of a new contributor to your project. Does your README clearly explain the purpose and function of your package without using jargon? Make any edits you like to make your README clear and welcoming.
+ 
+2. Create a Code of Conduct
+- Create a file in the top level of your package called CODE-OF-CONDUCT.md by running `usethis::use_code_of_conduct()`. This function automatically populates your code of conduct with the [Contributor Covenant](https://www.contributor-covenant.org/). You may keep or modify the template, or you may draft your own entirely. Make sure to attribute the source of any outside text you use or modify, and make sure the license of the text you use permits your use or modification. Here are some other example codes of conduct:
+    - [Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md)
+    - [Django Code of Conduct](https://www.djangoproject.com/conduct/)
+- Include a link to your Code of Conduct in your README.
+ 
+3. Create Contributing Guidelines
+- Create a file called CONTRIBUTING.md by running `usethis::use_tidy_contributing()`. This file will contain information for people interacting with your project about how they can use your package and/or how they can get involved. Here are some example Contributing Guidelines:
+    - [Atom](https://github.com/atom/atom/blob/master/CONTRIBUTING.md)
+    - [Data Carpentry Geospatial R lesson](https://github.com/datacarpentry/r-intro-geospatial/blob/master/CONTRIBUTING.md)
+    - FIXME: more examples
```

Comment:
The [tidyverse](https://www.tidyverse.org/contribute/) contributing information is interesting; might be useful to explain that technical info about contributing code is linked at the bottom.